src/minio_utils/minio_utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+This module contains utility functions to interact with Minio.
+"""
+
+import os
+
+from minio import Minio
+
+
+def get_minio_client() -> Minio:
+    """
+    Helper function to get the Minio client.
+
+    :return: A Minio client object
+    """
+    return Minio(
+        os.environ['MINIO_URL'].replace("http://", ""),
```

Comment:
here I'm wondering why we're just not excluding `http://` in the env var in rancher. Is there a reason for that?

Reply:
tested removing `http://` from docker-compose file. When I tried to read delta files, it took 10 minutes without returning anything. I am guessing it's needed for spark custom s3 endpoint config.